src/gui/editors/PianoRoll.cpp

```diff
@@ -4453,18 +4675,42 @@ PianoRollWindow::PianoRollWindow() :
 	eraseAction->setShortcut( Qt::SHIFT | Qt::Key_E );
 	selectAction->setShortcut( Qt::SHIFT | Qt::Key_S );
 	pitchBendAction->setShortcut( Qt::SHIFT | Qt::Key_T );
-
 	connect( editModeGroup, SIGNAL( triggered( int ) ), m_editor, SLOT( setEditMode( int ) ) );
 
-	QAction* quantizeAction = new QAction(embed::getIconPixmap( "quantize" ), tr( "Quantize" ), this );
+//Create a new clickbutton action, to the right on tollbar of previous button(s)
+	QAction* quantizeAction = 
+    new QAction(embed::getIconPixmap( "quantize" ), tr( "Quantize" ), this );
 	connect( quantizeAction, SIGNAL( triggered() ), m_editor, SLOT( quantizeNotes() ) );
 
+	QAction* humanizeVelAction = 
+    new QAction(embed::getIconPixmap( "humaVel" ), tr( "Humanize Selected Note-Velocity" ), this );
+	connect( humanizeVelAction, SIGNAL( triggered() ), m_editor, SLOT( humanizeNotesVel() ) );
+
+    QAction* humanizeLenAction = 
+    new QAction(embed::getIconPixmap( "humaPos" ), tr( "Humanize Selected Note-position" ), this );
+	connect( humanizeLenAction, SIGNAL( triggered() ), m_editor, SLOT( humanizeNotesLen() ) );
+
+	QAction* strumNotesUpAction = 
+    new QAction(embed::getIconPixmap( "strumUp" ), tr( "Strum Selected by Q-value" ), this );
+	connect( strumNotesUpAction, SIGNAL( triggered() ), m_editor, SLOT( strumNotesUp() ) );
+	
+    QAction* strumNotesDnAction = 
+    new QAction(embed::getIconPixmap( "strumDwn" ), tr( "Strum Selected by Q-value" ), this );
+	connect( strumNotesDnAction, SIGNAL( triggered() ), m_editor, SLOT( strumNotesDn() ) );
+
+
+//add physical buttons for each of the defined QActions, in this order
```

Comment:
If I may, related to my comment above, I'd like to suggest new text for these:

1) 'Humanize selected (velocity)'
2) 'Humanize selected (start time)'
3) 'Strum up'
4) 'Strum down'

Reply:
Adding to this, maybe renaming the pixmaps to `humanizeVel`, `humanizePos` and `strumDown` would make names more understandable.